CIScripts/Test/TestConfigurationUtils.ps1

```diff
@@ -74,7 +74,7 @@ function Enable-VRouterExtension {
            [Parameter(Mandatory = $true)] [string] $ForwardingExtensionName,
            [Parameter(Mandatory = $false)] [string] $ContainerNetworkName = "testnet")
 
-    Write-Output "Enabling Extension"
+    Write-Host "Enabling Extension"
```

Comment:
If the semantics of `Write-Output` fit us better than `Write-Host`, we can make a separate PR for changing it in the whole repo. Still, I'm not sure which one is more correct there.